Content.Shared/Humanoid/Prototypes/SpeciesPrototype.cs

```diff
@@ -128,4 +128,5 @@ public enum SpeciesNaming : byte
     FirstLast,
     FirstDashFirst,
     TheFirstofLast,
+    LastFirst,
```

Comment:
This should have a comment on it since it's an upstream file, so it won't be overwritten by an update

Something like "this is for [species]"